### DISCUSSION TITLE
Buttons for channel forward / backward would be helpful

### DISCUSSION THREAD

**Discussion by theJazzman42** [KejPi/AbracaDABra#84] [Ideas]
The title says it all.

**Comment by KejPi** [comment]
You can switch the channels by mouse wheel.

**Comment by theJazzman42** [comment]
I know. It'll be easier with forward and backward buttons.

**Comment by KejPi** [comment]
Not a big problem to implement it, but I do not know how to integrate it in UI in a good looking and consistent way :-( 
By the way, up/down arrows also work.

**Comment by KejPi** [comment]
Implemented in c30957a4ac9df1759f6b5188d99fb0ec8cce5b7f

<img width="894" alt="Snímek obrazovky 2023-06-17 v 18 58 53" src="https://github.com/KejPi/AbracaDABra/assets/6438380/b4571600-9b59-4f8d-a754-dd99ddc82198">

**Comment by theJazzman42** [comment]
Cool.
